sensu/config.go

```diff
@@ -72,6 +73,15 @@ func NewConfigFromFile(flagset *configFlagSet, configFile string) (*Config, erro
 		return nil, errNoClientName
 	}
 
+	// If we reached this point, we have a client name.
+	// Let's also add the default subscription - client:name
+	// Without at least one subscription sensu server will crash.
+	cfg.config.Client.Subscriptions = append(cfg.config.Client.Subscriptions,
```

Comment:
if you need at least one subscriptions  you don't need to add the value all the time  just ensure `len(cfg.config.Client.Subscriptions) == 0`